src/manager.cpp

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
The Meson changes required to compile this file are missing. I left a comment about this in the previous commit.